queue.go

```diff
@@ -22,6 +23,18 @@ var (
 	publisherWid uint64 = 0
 )
 
+const urlString = "amqp://%s:%s@%s:%s/%s"
```

Comment:
Why this is global? It is used only in the `NewQueue()` so move it there

Reply:
done